lib/Exporter/ONNXModelWriter.cpp

```diff
@@ -691,6 +691,18 @@ ONNXModelWriter::writeSparseToDenseMask(const SparseToDenseMaskNode *node,
   return writeAllWithNode(node->getName(), node, proto);
 }
 
+llvm::Error
+ONNXModelWriter::writeAdaptiveAvgPool(const AdaptiveAvgPoolNode *node,
```

Comment:
We still need this?

Reply:
Yeah actually this just landed yesterday #3167